### ISSUE TITLE
how to get the class label from output data

### ISSUE THREAD

**Issue by pango99** [faedtodd/Tensorrt-yolov3-win10#2]
Hi,in postProcess() function,you extract the score & rect infor from output data, but how to get the class label info if my model has multi class?

**Comment by XuHao9166** [comment]
> Hi,in postProcess() function,you extract the score & rect infor from output data, but how to get the class label info if my model has multi class?

do you finish this problem?how to get the class label?

**Comment by pango99** [comment]
> 
> 
> > Hi,in postProcess() function,you extract the score & rect infor from output data, but how to get the class label info if my model has multi class?
> 
> do you finish this problem?how to get the class label?

No,I still dont know how to get,now I'm using caffe version yolo

**Comment by XuHao9166** [comment]
> > > Hi,in postProcess() function,you extract the score & rect infor from output data, but how to get the class label info if my model has multi class?
> > 
> > 
> > do you finish this problem?how to get the class label?
> 
> No,I still dont know how to get,now I'm using caffe version yolo
caffe version yolo is good choose,but it hard to train model. i only use pytorch or darknet . how you train caffe model by yourself? i means train and change to onnx.

**Comment by pango99** [comment]
> 
> 
> > > > Hi,in postProcess() function,you extract the score & rect infor from output data, but how to get the class label info if my model has multi class?
> > > 
> > > 
> > > do you finish this problem?how to get the class label?
> > 
> > 
> > No,I still dont know how to get,now I'm using caffe version yolo
> > caffe version yolo is good choose,but it hard to train model. i only use pytorch or darknet . how you train caffe model by yourself? i means train and change to onnx.

I train model using darknet,and then convert yolo to caffe,so the model can be accelerated by tensorrt;the convert script can be found on github

**Comment by ghost** [comment]
I solve it with 

int max_index = -1;
float max_value = FLT_MIN;
for (int k = 5; k < CATEGORY + 5; k++) {
    if (ptr[k] > max_value) {
        max_value = ptr[k];
        max_index = k - 5;
    }
}
